### ISSUE TITLE
Permission denied to access property "addEventListener" on cross-origin object

### ISSUE THREAD

**Issue by riccardo-forina** [SimeonC/storybook-xstate-addon#30]
I'm deploying my Storybook with Chromatic. I noticed that if I use this addon, Chromatic's library breaks for all stories, even those without the inspector enabled.

Chromatic loads the story's iframe in a webapp hosted under www.chromatic.com, but the iframe lives in a different domain (xxx.chromatic.com). I guess they are setting a strict sandbox.
Opening the iframe's src in a new tab works just fine.

It would be nice to detect/catch those errors and handle them gracefully so not to break Chromatic's library. 

Full error stack following
```
withXstateInspector/<@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:2657444
triggerEffects/<@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:331359
triggerEffects@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:331273
HooksContext/this.renderListener@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:330137
handleEvent/<@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:730987
handleEvent@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:730962
handler@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:729669
emit@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:729724
_callee3$@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:1911574
tryCatch@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:914659
invoke@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:914290
defineIteratorMethods/</<@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:915434
asyncGeneratorStep@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:1901145
_next@https://xxx.chromatic.com/vendors~main.2998908c.iframe.bundle.js:2:1901465
```

**Comment by SimeonC** [comment]
:rocket: Issue was released in [`v2.3.6`](https://github.com/SimeonC/storybook-xstate-addon/releases/tag/v2.3.6) :rocket:

**Comment by SimeonC** [comment]
:rocket: Issue was released in [`v2.3.6`](https://github.com/SimeonC/storybook-xstate-addon/releases/tag/v2.3.6) :rocket:
